tools/pipelines/templates/include-test-real-service.yml

```diff
@@ -141,16 +141,6 @@ stages:
           value: true
         - name: artifactPipeline
           value: Build - client packages
-        - name: feed
-          ${{ if eq(variables.isTestBranch, 'true') }}:
-            value: $(ado-feeds-internal) # Comes from the ado-feeds variable group
-          ${{ else }}:
-            value: $(ado-feeds-build) # Comes from the ado-feeds variable group
-        - name: devFeed
-          ${{ if eq(variables.isTestBranch, 'true') }}:
-            value: $(ado-feeds-internal) # Comes from the ado-feeds variable group
-          ${{ else }}:
-            value: $(ado-feeds-dev) # Comes from the ado-feeds variable group
```

Comment:
`feed` was used for the `@fluidframework` and `@fluid-experimental` scopes. `devFeed` for `@fluid-internal`, `@fluid-private`, and `@fluid-tools`. They are all now available through the new `$(ado-feeds-ff-download-only)` feed.